README.md

```diff
@@ -10,6 +10,104 @@ Centos 6
+`:owner`                | String    | Owner of application files (user)             | `chef`
+`:group`                | String    | Owner of application files (group)            | `chef`
+`:repository`           | String    | URL to git repository                         | `nil`
+`:revision`             | String    | Branch or Commit Hash of Repo                 | `master`
+`:virtualenv_path`      | String    | Path to python virtualenv                     | `/opt/venv_#{ project_name }`
+`:config_template`      | String/Nil| Path to app config template                   | `settings.py.erb`
+`:config_destination`   | String/Nil| Path to app config location on server         | `nil`
+`:config_vars`          | Hash      | Variables for the config template             | `nil`
+`:requirements_file`    | String/Nil| Application's dependencies file.              | `setup.py`
+`:django_migrate`       | Boolean   | Whether or not to run django migrations       | `false`
+`:django_collectstatic` | Boolean   | Whether or not to run django collectstatic    | `false`
+`:interpreter`          | String    | Python command (python, python3, etc)         | `python`
+`:gunicorn_port`        | Int/Nil   | Port to run gunicorn on                       | `nil`
+
+For more information on some of thesse resources see the 'Notes' section below.
+
+### Example
+
+**SETUP:** Before you can use the python-webapp cookbook add the following
+informtion to the corresponding files:
+
+* Add `depends 'python-webapp'` to your metadata.rb.
+* Add `cookbook 'python-webapp', git:
+  'git://github.com/osuosl-cookbooks/python-webapp.git', branch: 'master'` to your
+  Berksfile
+
+**RECIPE USAGE:** The following code would go in a `some_recipe.rb` file:
+
+```
+    path = '/opt/test_app'
+
+    python-webapp 'test_app' do
+      create_user true
+      owner 'test_app'
+      group 'test_app'
+
+      repository 'https://github.com/osuosl/python-test-apps.git'
+      revision 'develop'
+
+      config_template 'config.yml.erb'
+      config_destination "#{path}/config.yml"
+      config_vars(
+          path: path,
+          engine: 'django.db.backends.sqlite3',
+          dbname: "#{path}/yourdatabasename.db"
+      )
+      interpreter 'python2.7'
+
+      django_migrate true
+      django_collectstatic true
+
+      gunicorn_port 8888
+    end
+```
+
+In the above example `test_app` should be the name of your python application.
+
+More examples of the python-webapp's usage can be found in this repo's `test`
+directory at
+https://github.com/osuosl-cookbooks/python-webapp/tree/master/test/cookbooks/python-webapp-test
+
+### Notes
+
+**python versions:** If you are using a version of python that does not come by
+default on your system you have to install this seperately; python-webapp will
+not automatically install a given version of python for your system.
+
+**wsgy.py:** One assumption made by python webapp is that if you are running a
```

Comment:
Uhhh I'm not sure this is true. We have zero Flask support right now.
Also it's wsgi for 'Web Server Gateway Interface'.